content/uiux/concepts/ux-psychology/terms/jacobs-law/jacobs-law.md

```diff
@@ -0,0 +1,29 @@
+---
+Title: "Jacob's Law"
+Description: "Users prefer your site to work like others they already know, because they spend most of their time on other sites."
```

Comment:
Please avoid using `""` instead of `''` to keep the content consistent with other doc entries.

```suggestion
Description: 'Jacobs Law states that users expect new interfaces to work like the ones they already know, making familiarity essential for a smooth experience.'
```